articles/tutorials/building_2d_games/04_working_with_textures/index.md

```diff
@@ -0,0 +1,344 @@
+
+After adding the *logo.png* file, you're project node should look similar to the following:
+
+<figure><img src="./images/mgcb-logo-added.png" alt="Figure 4-6: Logo image added to the MGCB Editor"><figcaption><p><strong>Figure 4-6: Logo image added to the MGCB Editor</strong></p></figcaption></figure>
+
+Save the changes by pressing `CTRL+S`, or by clicking the *Save* icon in the top tool bar, or by choosing *File > Save* from the top menu. When you save the changes made, the MGCB Editor will write updates to the *Content.mgcb* file in your project to reflect the changes made. After saving, you can close the MGCB Editor.
+
+### Loading the Image Using [ContentManager](xref:Microsoft.Xna.Framework.Content.ContentManager)
+
+With the image now added to the content project, whenever a build of our game project is performed, the content pipeline will automatically handle compiling the image into an optimized format and ensuring that the compiled asset is copied to the final game project build directory.  This process is handled by the *MonoGame.Content.Builder.Tasks* NuGet package reference.  Now we need to update our game to load the asset.
+
+First, open the *Game1.cs* file in your project and add
+
+```cs
+private Texture2D _logo;
+```
+
+beneath where the [GraphicsDeviceManager](xref:Microsoft.Xna.Framework.GraphicsDeviceManager) and [SpriteBatch](xref:Microsoft.Xna.Framework.Graphics.SpriteBatch) instance member variables are declared.  This adds a new [Texture2D](xref:Microsoft.Xna.Framework.Graphics.Texture2D) instanced called `_logo`.  [Texture2D](xref:Microsoft.Xna.Framework.Graphics.Texture2D) is the type used to store a reference to 2D image data in MonoGame.
+
+Next, locate the [LoadContent](xref:Microsoft.Xna.Framework.Game.LoadContent) method and add the following after the `_spriteBatch` is instantiated:
+
+```
+_logo = Content.Load<Texture2D>("images/logo");
+```
+
+This uses the [ContentManager](xref:Microsoft.Xna.Framework.Content.ContentManager) object provided by the [Content](xref:Microsoft.Xna.Framework.Game.Content) property of the [Game](xref:Microsoft.Xna.Framework.Game) class to load the compiled image file.  We use the [Load<T>](xref:Microsoft.Xna.Framework.Content.ContentManager.Load``1(System.String)) method where `T` is the content type that we are loading, in this case a [Texture2D](xref:Microsoft.Xna.Framework.Graphics.Texture2D).  The parameter we pass to it is the *asset name*, which is a string representation of the path to the content to load, without any extensions.  The path used as the asset name is also relative to the value of [Content.RootDirectory](xref:Microsoft.Xna.Framework.Content.ContentManager.RootDirectory) which was set to `Content` in the `Game1` constructor.  THis means the asset name of our logo image located at *Content/images/logo.png* is `images/logo`.

```

Comment:
Typo "THis means"
Although I miss the days when you could copy the name of the file from the content pipeline editor :S

Reply:
Typo corrected